eloquentjs/2_3_chessboard.js

```diff
@@ -0,0 +1,11 @@
+function Chessboard(width, height){
+    for(let i = 0; i <height; i++){
+        let output='';
+        for(let j=0;j<width;j++){
+            output += (j%2!=i%2) ? '#': ' ';
```

Comment:
This line is somewhat confusing, but looks correct. It's fine the way it is, but it may be worth considering if there are ways to make it clearer for readers.